.env.template

```
@@ -7,3 +7,8 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY=<TRELLO_API_KEY>
+TRELLO_API_TOKEN=<TRELLO_API_TOKEN>
+TRELLO_BOARD_ID=<TRELLO_BOARD_ID>
+TRELLO_LIST_ID=<TRELLO_LIST_ID>
```

Your Readme instructions are clear, but guessing this was just an earlier iteration
```suggestion
TRELLO_TODO_LIST_ID=<TRELLO_TODO_LIST_ID>
TRELLO_DONE_LIST_ID=<TRELLO_DONE_LIST_ID>
```